### ISSUE TITLE
Items

### ISSUE THREAD

**Issue by SqueeG** [SqueeG/awesomeTome#25]
Are we going to be using Red Rob's work on magic items?

**Comment by Tarkisflux** [comment]
I thought that we were using Surgo's from the wiki, but I can re-read Red Rob's and generate an opinion on it. It's even timely, since he just completed a whole bunch of edits and revamps to it and I can now go swimming in current text...

Should have that done by later today, after I do a bit of editing elsewhere.

**Comment by SqueeG** [comment]
I just have a terrible memory and saw his revised content, so I asked.

**Comment by ExplosiveRunes** [comment]
I was actually going to bring this up too, having read Red Robs update earlier. Red Rob's Tome-style magic item rules seem to be much more comprehensive than Surgo's, it's equivalent to a full magic item chapter like you'd see in the DMG. Granted, it has less specific magic items than the wonderous item section of the SRD or DMG, but the underlying system is there, and I think it would be simple to convert some, or write up new ones wholesale.

**Comment by Tarkisflux** [comment]
Yeah, that's the impression I'm getting while I look it over. It is much better integrated as a "treasure" and economy chapter and I agree that it's a better fit for the project overall. 

There's a lot of changes in there from Tome-standard along with a lot of additions that may have interactions we don't like though (attribute growth changes, limits on active harmless spells, moderate and greater scrolls, scroll cast time increase, other things I'm finding while I look it over), and I may want to kick a few of those things back at him after I finish the thread and his new one (so I don't kick back things that have been explained or already kicked back). If you wanted to start transcribing it you could, but if you just wanted to convert from bbcode it might be worth it to wait rather than try to find potentially small changes later on.

**Comment by ExplosiveRunes** [comment]
Even if we do decide to use it, which I'm in favor of, I'd want to wait at least a month until it gets finalized. I know from personal experience that when I post a large thing like that, I usually end up editing it several times, over a span of time.

**Comment by ExplosiveRunes** [comment]
I've formatted and included red robs mundane armors as a precursor to doing the magic item chapter, which I'll probably hold off on until red rob does his next revision. I'll probably edit the armor entries for tone (switching out abbreviations for the words, etc) without changing the mechanics. One exception, I feel we should change armors that "Prevent an amount of [type of damage] damage" to give DR instead so that they properly interact with abilities that ignore damage reduction.

@Tarkisflux I'd appreciate it if you gave the formatting a look over.
